packages/ember-data/lib/system/store/finders.js

```diff
@@ -29,13 +29,11 @@ export function _find(adapter, store, typeClass, id, record) {
       return store.push(typeClass, payload);
     });
   }, function(error) {
-    var record = store.getById(typeClass, id);
```

Comment:
This line has been added in [63795a4](https://github.com/emberjs/data/commit/63795a40c833eb23b6f3e78841b65bee4418278c#diff-1636bea36bbe333a834ea11fbfbde48dR1385), where the `record` parameter was not available. I guess that's the reason why a `var record = store.getById(typeClass, id)` has been used here; the `record` parameter is added in [51728a8](https://github.com/emberjs/data/commit/51728a806426309f494de5d8930b9f8951ffac6a#diff-1636bea36bbe333a834ea11fbfbde48dR1869). So I think reusing the param down below is save...